daemon/cmd/status.go

```diff
@@ -307,6 +307,9 @@ func (d *Daemon) getKubeProxyReplacementStatus() *models.KubeProxyReplacement {
 			features.NodePort.Algorithm = models.KubeProxyReplacementFeaturesNodePortAlgorithmMaglev
 			features.NodePort.LutSize = int64(option.Config.MaglevTableSize)
 		}
+		if option.Config.LoadBalancerAlgConf == option.LoadBalancerAlgAnnotation {
+			features.NodePort.LutSize = int64(option.Config.MaglevTableSize)
+		}
```

Comment:
can be follow-up, but would be nice to reflect the config as part of the cilium status --verbose so that it clearly states global vs annotation.

Reply:
It's basically extending the openapi yaml/model and then populating the data with it. I can take care of it as follow-up once we land this no problem.

Reply:
thanks for the info. I can take care of it later :)